app/controllers/PayPal.scala

```diff
@@ -47,6 +51,47 @@ class PayPal(
     }.map(token => Ok(Token(token).asJson))
   }
 
+  // scalastyle:off parameter.number
+  def execute(
+    paymentId: String,
+    token: String,
+    payerId: String,
+    CMP: Option[String],
+    INTCMP: Option[String],
+    refererPageviewId: Option[String],
+    refererUrl: Option[String],
+    pvid: Option[String],
+    bid: Option[String],
+    ophanVisitId: Option[String],
+    componentId: Option[String],
+    componentType: Option[ComponentType],
+    source: Option[AcquisitionSource],
+    refererAbTest: Option[AbTest],
```

Comment:
Is there any need for the AbTest to be deserialised from JSON and then serialised again?  Could this not just be passed through as a string?

That way if the structure of AbTest change support.frontend won't need to be updated.

Reply:
That's a nice point. I could actually make all of these types strings, and let the API (currently contributions-frontend, soon to be the new payments API that Joe is scoping out) deal with any incorrect fields